benchmark_utils/base_solver.py

```diff
@@ -238,3 +243,13 @@ def get_result(self):
             cv_results=self.cv_results_,
             dict_estimators=self.dict_estimators_
         )
+
+
+class DeepDASolver(DASolver):
+    n_jobs = 1
+
+    def __init__(self, **kwargs):
+        super().__init__(print_infos=False, **kwargs)
```

Comment:
Why here `print_infos=False`? Feel like more logs, the better

Reply:
When `DASolver` or `DeepDASolver` are inherited, `n_jobs` and `device` are printed. But `n_jobs` is overwritten by `DeepDASolver`. In the end, `n_jobs` and `device` are always printed but the `print_infos` argument makes sure that the right value is printed.